### ISSUE TITLE
peeringdb-py inconsistent PoC data

### ISSUE THREAD

**Issue by ccaputo** [peeringdb/peeringdb#1633]
With a fresh peeringdb-py generated sqlite3 database, data which shows up in https://www.peeringdb.com/api/poc when viewed from a logged in account, differs from that which make it into the peeringdb-py sqlite3 database.

For example, a search for a phone number that ends with 664638 will show up 5 times when viewing https://www.peeringdb.com/api/poc and zero times when the following is performed:

```sqlite3 peeringdb.sqlite3 .dump | grep -i 664638```

This phone number has the visibility setting of "Users" and thus should be making it to the peeringdb-py database, but it doesn't. Other PoC data with a visibility setting of "Users" does (presuming an api-key-based peeringdb-py sync).

It is appears there is a bug that is preventing peeringdb-py users from seeing the data they would expect to see.

**Comment by jwallskog** [comment]
+1
This is hindering our ability to automate using peeringdb-py. Could you prioritize this request?

**Comment by jackcarrozzo** [comment]
Just clarifying, this previously worked as expected?

**Comment by ccaputo** [comment]
> Just clarifying, this previously worked as expected?

I think so. I have an old copy (circa January 2024) of the sqlite3 database produced by peeringdb-py and it does contain the data referenced in my example above.

**Comment by grizz** [comment]
This definitely used to work, so clear bug.

**Comment by grizz** [comment]
@leovegoda I think this should be a high prio fix.

**Comment by jackcarrozzo** [comment]
+1

**Comment by martinhannigan** [comment]
+1

**Comment by leovegoda** [comment]
@ccaputo We'll try to get this into 2.60.0, the next release

**Comment by ccaputo** [comment]
Turns out that the flag `--fetch-private` is needed for an initial database sync and subsequent syncs, if private data is needed. https://docs.peeringdb.com/howto/peeringdb-py/ has been updated accordingly.

I have tested this flag. Based on the results, I believe this issue can be closed.

The fix for those affected is as follows: 1) Delete your existing sqlite3 database. 2) Update your scripts to include the `--fetch-private` flag with all `peeringdb sync` commands, ala "`peeringdb sync --fetch-private`".

Note that you will need to have valid authentication set up for this (preferably with an API key) in order for private data to be fetched.
